About
=====

gutSMASH is an algorithm that enables to systematically evaluate the metabolic potential of anaerobic bacteria by predicting both known and putative Metabolic Gene Clusters (MGCs). This tool is specifically designed to uncover MGCs that are involved in specialized primary metabolism from the gut microbiome that have been associated to microbe-microbe and host-microbe interactions. Moreover, gutSMASH also predicts energy-capturing-related MGCs, to evaluate the mechanisms bacteria employ to harvest energy. To be able to detect these MGCs, gutSMASH uses detection rules logic as antiSMASH, that are Pfam combination defining of a specific reaction. For more information on how to use gutSMASH you can also visit our online documenation in: https://gutsmash.bioinformatics.nl/help.html


How to run gutSMASH
^^^^^^^^^^^^^^^^^^^

 - You can try out gutSMASH by using our online webserver: https://gutsmash.bioinformatics.nl/
 - or download and install the gutSMASH command line version from here: https://github.com/victoriapascal/antismash/tree/gutsmash

How to cite
^^^^^^^^^^^

If you have found gutSMASH useful, please cite our preprint on the algorithm: https://www.biorxiv.org/content/10.1101/2021.02.25.432841v1, and/or the article describing the gutSMASH web server: https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkab353/6279837 

